models/us_census_acs/code/upload.py

```
@@ -0,0 +1,119 @@
+        path=str(path),
+        source_format="parquet",
+        if_table_exists="replace",
+        if_storage_data_exists="replace",
+        if_dataset_exists="pass",
+        chunk_size=100
+        * 1024
+        * 1024,  # resumable 100MB chunks -> resilient to SSL drops on large files
+    )
+
+    client = bigquery.Client(project=BILLING_PROJECT)
+    q = f"select count(*) as n from `{BILLING_PROJECT}.{DATASET_ID}_staging.{slug}`"
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)